torch/fx/node.py

```diff
@@ -628,6 +628,15 @@ def maybe_replace_node(n : Node) -> Node:
         assert isinstance(new_kwargs, dict)
         self.__update_args_kwargs(new_args, new_kwargs)
 
+    def rename(self, candidate: str):
+        if candidate == self.name:
+            return
+        name = self.graph._graph_namespace.create_name(candidate, None)
+        self.name = name
+        assert self in self.graph._graph_namespace._obj_to_name
+        self.graph._graph_namespace._obj_to_name[self] = name
+        self.graph._graph_namespace._used_names.add(name)
```

Comment:
conversely, these manipulations on graph namespace seem like they should validly live on Namespace (and BC is not a problem there since it's a private API)

Reply:
I'll move it there